crates/weaver_forge/expected_output/test/span/cosmosdb.md

```diff
@@ -20,7 +20,7 @@ An identifier for the database management system (DBMS) product being used. See
   
 - Tag: connection-level
   
-- Type: Enum [other_sql, mssql, mssqlcompact, mysql, oracle, db2, postgresql, redshift, hive, cloudscape, hsqldb, progress, maxdb, hanadb, ingres, firstsql, edb, cache, adabas, firebird, derby, filemaker, informix, instantdb, interbase, mariadb, netezza, pervasive, pointbase, sqlite, sybase, teradata, vertica, h2, coldfusion, cassandra, hbase, mongodb, redis, couchbase, couchdb, cosmosdb, azure.cosmosdb, dynamodb, neo4j, geode, elasticsearch, memcached, cockroachdb, opensearch, clickhouse, spanner, trino]
```

Comment:
this cosmosdb friend is marked with annotation to exclude it from codegen, so it should never been generated here